src/preview.ts

```diff
@@ -56,10 +63,8 @@ export class Preview extends TypedEventEmitter<Preview.Events> {
   }
 
   private async launch() {
-    const localDataDir = await tmpNamePromise({ prefix: 'marp-cli-carlo-' })
-
     this.carloInternal = await carlo.launch({
-      localDataDir,
+      localDataDir: path.resolve(os.tmpdir(), './marp-cli-carlo'),
```

Comment:
Update the temporary directory to use a fixed path to prevent random failing when finalization.